lib/actions/device.coffee

```diff
@@ -186,6 +186,70 @@ exports.reboot =
 		resin = require('resin-sdk')
 		resin.models.device.reboot(params.uuid).nodeify(done)
 
+exports.enableDeviceUrl =
+	signature: 'device enableDeviceUrl <uuid>'
```

Comment:
Maybe we can use more human-like command names to be consistent with the other commands? Like `device enable url <uuid>`? Notice you can freely use multi-words commands.

Also, the device in `deviceUrl` is probably redundant with the initial `device` command namespace?